labs/firepower-restapi-105/2.md

```diff
@@ -1,22 +1,22 @@
 ## Problem: REST operation fails with HTTP 403
-This may due to many access and privileges-related reasons.
+This problem may be due to many access and privileges-related reasons.
 
-Login into the FMC UI via [your own setup](https://hostnameorIPaddressofyourFMC) if you are using the [DevNet Sandbox](https://fmcrestapisandbox.cisco.com).
+Login into the FMC UI via your own setup (example: https://hostnameorIPaddressofyourFMC) if you are using the [DevNet Sandbox](https://fmcrestapisandbox.cisco.com).
 
-Use the same user credentials, whoever's authentication token you are using for API. Attempt the same operation using UI. If it fails, then you need to request your admin to change access privileges of the user.
+Make sure you use the same user credentials for logging in as you use to request an authentication token for the API. Attempt the same operation using UI. If it fails, then you need to request your admin to change access privileges of the user.
 
 Make sure the URL in the request has the right **domain** in which the *user* is authorized.
 
 **Note:** DevNet Sandbox may have restrictions on privileges.
 
 
 ## Problem: A REST operation fails with http 401
-Most likely the authentication token you are using for the API has expired. You have to refresh the token. Please read more on refresh token in the [Learning Lab FMC REST API 102](https://learninglabs.cisco.com/lab/ucsd-rest-api-102/step/1).
+Most likely the authentication token you are using for the API has expired. You have to refresh the token. Please read more about refreshing tokens in the [Learning Lab FMC REST API 102](https://learninglabs.cisco.com/lab/ucsd-rest-api-102/step/1).
```

Comment:
I think this link should go to https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1 instead of the UCSD link.